### PR TITLE
Instructions for building a rover

### PR DESCRIPTION
**Pull request by aninternetof**
Here is my first pass at instructions for building a rover.

Some things are not described in detail enough for a novice to succeed (e.g. rebuilding the kernel with the UVC driver). Hopefully, those steps will be circumvented in the future, so I didn't put a ton of effort into their instructions.

Let me know if you do see any major gaps, though!

### CODE REVIEW

File: www/docs/build-a-rover.rst [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+- Depending on what version of the Thunder Tumbler you happen to get, the radio controller IC could be through-hole or surface-mount. If it's surface mount, try your best not to rip off the pads when you remove the IC.
+- To figure out which pads are the variable left/right forward/backward motor pads, I recommend connecting a wire to the 3.3V supply on your C.H.I.P, then poking the other end around on all the pads. Observe which wheel turns and in which direction, and write it down!
+- You can get ground from anywhere you want; you don't need to use a pad from the radio control IC. Use a multimeter to find a spot that reads zero resistance with the negative terminal of the RC car's battery holder.
+
+Connecting to the C.H.I.P
+++++++++++++++++++++++++++
+
+Use your jumpers to connect the signals on your new 5-pin header to the C.H.I.P.
+Ground connects to ground, and the motor control signals can connect to any
+`XIO-P[0-7]` pin. Right now the pins are hard-coded iZn `blockly-api.js` (booooo, I
```
Comment (cabarnes) [inline www/docs/build-a-rover.rst]: `in`
Also, you might want to link to `blockly-api.js` here.

File: www/docs/build-a-rover.rst [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+- ground
+- signal (this is what varies to indicate something is detected)
+
+We just want a binary signal out of the sensors, so even though we have a
+continuous analog signal coming out of the sensor, we won't hook it up to
+an analog input of the C.H.I.P. We'll just hook it up to a regular GPIO input,
+and let the input hardware of the pin serve as a rough comparator.
+
+Just like the motor signal pins, the pins for the IR ear signals are hardcoded
+at the moment (this time in `app.py` -- we are really gonna make this configurable
```
Comment (cabarnes) [inline www/docs/build-a-rover.rst]: You might want to link to `app.py` here.

File: www/docs/build-a-rover.rst [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+| right             | XIO-P4      |
++-------------------+-------------+
+
+Note: These sensor circuits are not great. Their detection range is only of a couple of inches.
+Our future reference design will include `a PCB with a Silicon Labs I2C MEMS
+IR sensor <https://upverter.com/ductape/aef33f7c39fd29d5/rovercode-prox-sensor/>`_, which should work much better.
+
+webcam -- the eye
+-------------------
+:important note: The default CHIP kernel does not enable the USB Video Class (UVC) driver. In the future we hope to provide a ready-to-use emmc image with this driver included, but for now you'll have to rebuild the kernel with the UVC driver included. This is a more advanced task. Your best bet is `this tutorial <http://www.raspibo.org/wiki/index.php/Compile_the_Linux_kernel_for_Chip:_my_personal_HOWTO>`_. If you're not up for this, don't worry -- just stay tuned for an update to this page telling you where you can get a ready-to-use emmc image.
```
Comment (cabarnes) [inline www/docs/build-a-rover.rst]: `eMMC`

File: www/docs/build-a-rover.rst [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+
+install rovercode service
+--------------------------
+Connect to the C.H.I.P. via serial or SSH.
+
+Follow the Standard Setup on the `quickstart page <quickstart.html>`_.
+
+play
+------
+Go to `<rovercode.com>`_, sign up for an account, then go to `Mission Control
```
Comment (cabarnes) [inline www/docs/build-a-rover.rst]: I think you need the `https://` here for the link to work correctly for `rovercode.com`.

File: www/docs/build-a-rover.rst (hunk 1 above)
Comment (aninternetof) [inline www/docs/build-a-rover.rst]: Good idea. Fixing.

File: www/docs/build-a-rover.rst (hunk 2 above)
Comment (aninternetof) [inline www/docs/build-a-rover.rst]: Good idea. Adding.

**Pull request merged**